### DISCUSSION TITLE
Making private GitHub Pages sites on Team tier

### DISCUSSION THREAD

**Discussion by nebbles** [community/community#10807] [Enterprise]
I was disappointed to find out this feature, which would be really valuable to our team in publishing internal documentation, is only available to GitHub Enterprise Cloud[^1].

Are there any plans to expand this to other paid tiers? We're a startup, so it's a bit excessive for us to be moving to an Enterprise tier already.

[^1]: https://docs.github.com/en/pages/getting-started-with-github-pages/changing-the-visibility-of-your-github-pages-site

**Comment by mmarkov** [comment]
We are small startup team of 9 engineers and it was a huge frustration to learn that private github pages are not available in Team plan. It would be great to have it with some reasonable limitations as now we are forced to solve our needs in other way which disincentivize any future upgrades as our team will grow, as we will have workaround in place already.

**Comment by ahill187** [comment]
Yes, I agree. The cost difference from Team to Enterprise is quite significant, and I don't need a lot of the Enterprise features at the current time. I'd be willing to pay a bit extra for an add-on to Team that allows private GitHub Pages.
